public/js/main.js

```diff
@@ -200,3 +200,90 @@ async function updateAddress() {
   }
   
 }
+
+// Upload Photo from barista profile to Cloudinary 
+let signData = new Object()
+let cloudinary_url = new String()
+
+async function fetchSignature() {
+  const signResponse = await fetch('/barista/signuploadform', {
+    method: "get"
+  });
+  signData = await signResponse.json();
+
+  cloudinary_url = "https://api.cloudinary.com/v1_1/" + signData.cloudname + "/auto/upload";
+    console.log(signData)
+}
+
+async function uploadProfilePhoto(){
+  try{
+    const files = document.getElementById("uploadPhoto").files;
+    const formData = new FormData();
+
+    for (let i = 0; i < files.length; i++) {
+      let file = files[i];
+      formData.append("file", file);
+      formData.append("api_key", signData.apikey);
+      formData.append("timestamp", signData.timestamp);
+      formData.append("signature", signData.signature);
+      formData.append("eager", "c_thumb,h_150,w_150,g_face");
+      formData.append("folder", "profile_photos");
+    }
+    
+    const photoDataResponse = await fetch(cloudinary_url, {
+      method: "POST",
+      body: formData
+    })
+    const photoData = await photoDataResponse.json()
+    console.log(photoData)
+
+    // const deletePrevPhoto = await fetch()

```

Comment:
I'd recommend being a little more diligent about removing commented code and `console` statements. If you leave them in, there should usually be a reason you're doing so, and there should be a comment explaining why.